Dockerfile

```diff
@@ -0,0 +1,106 @@
+# Build:
+#   docker build -t discourse/ruby:3.3-bookworm-slim --build-arg DEBIAN_RELEASE=bookworm --build-arg RUBY_VERSION=3.3.3 .
+#
+# You are not expected to build and push this Docker image by hand. This is done by the `build` Github Actions workflow.
+ARG DEBIAN_RELEASE=
+ARG RUBY_VERSION=
+
+FROM debian:${DEBIAN_RELEASE}-slim
+
+ARG RUBY_VERSION
+
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US.UTF-8
+ENV LC_ALL en_US.UTF-8
+ENV RUBY_VERSION=${RUBY_VERSION}
+
+# Installs system dependencies required to run Ruby
+RUN set -eux; \
+  DEBIAN_FRONTEND=noninteractive apt-get update; \
+  DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+  bzip2 \
+  ca-certificates \
+  libgmp-dev \
+  libssl-dev \
+  libyaml-dev \
+  procps \
+  zlib1g-dev \
+  ; \
+  rm -rf /var/lib/apt/lists/*; \
+  # skip installing gem documentation
+  mkdir -p /usr/local/etc; \
+  { \
+  echo 'install: --no-document'; \
+  echo 'update: --no-document'; \
+  } >> /usr/local/etc/gemrc; \
+  # Installs the dependencies required to build Ruby
+  savedAptMark="$(apt-mark showmanual)"; \
+  DEBIAN_FRONTEND=noninteractive apt-get update; \
```

Comment:
We just called `apt-get update` earlier, this should not be necessary (if we remove the `rm -rf /var/lib/apt/lists/*` above).

Reply:
Good catch.